services/gateway-api/src/circuit_breaker/test_circuit_breaker.py

```
@@ -0,0 +1,538 @@
+        )
+    )
+
+    # Open circuit
+    async def failing_call():
+        raise Exception("API error")
+
+    with pytest.raises(Exception):
+        await breaker.call(failing_call)
+
+    assert breaker.is_open
+    first_timeout = breaker._opened_at
```

Ruff (0.14.8)

Local variable `first_timeout` is assigned to but never used

Remove assignment to unused variable `first_timeout`

(F841)